music_this_week_app/backend/spotifyHandler.py

```diff
@@ -55,10 +60,12 @@ def filter_artist(self, artist_name):
             except spotipy.client.SpotifyException as error:
                 print("ERROR: Failed to search twice. Error below:")
                 print(error)
+                self.save_new_artist(artist_name, None)
```

Comment:
My bad, I thought those errors were for a lack of results due to that artist not being on spotify and that your conditional below was to handle some weird edge case.

Certainly can refactor this.